bbp/default.nix

```diff
@@ -221,6 +221,12 @@ let
         bluepy = callPackage ./nse/bluepy {
         };
 
+        bluepy_0_11_2 = callPackage ./nse/bluepy {
```

Comment:
Yes, that version is deprecated AFAIK. But I didn't want to break it, in case somebody is using it. We plan to make 0_11 the new default version in the future.